app/views/layouts/mailer.html.erb

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <style>
+      /* Email styles need to be inline */
```

Comment:
might be good to have a style or two for the email? that can be a next step i guess